express/blocks/floating-buttons/floating-buttons.js

```diff
@@ -0,0 +1,68 @@
+import BlockMediator from '../../scripts/block-mediator.min.js';
+import { formatDynamicCartLink } from '../../scripts/utils/pricing.js';
+
+function initScrollWatcher(block) {
+  const hideOnIntersect = new IntersectionObserver((entries) => {
+    const notIntersecting = entries.every((entry) => !entry.isIntersecting);
+
+    if (notIntersecting) {
+      block.classList.remove('hidden');
+    } else {
+      block.classList.add('hidden');
+    }
+  }, {
+    root: null,
+    rootMargin: '32px',
+    threshold: 0,
+  });
+
+  const footer = document.querySelector('footer');
+  if (footer) hideOnIntersect.observe(footer);
+
+  const primaryCta = BlockMediator.get('primaryCtaUrl');
+  if (!primaryCta) return;
+
+  const primaryUrl = new URL(primaryCta);
+
+  const pageCta = Array.from(document.querySelectorAll(
```

Comment:
I think this should also search for con-buttons now , since those are normal milo button class names. 
I had noticed on the nonprofit page that the button only appears when a user scrolls down, and this is not currently working. - https://www.adobe.com/express/nonprofits
If you add a check for .con-button I imagine it'll then work.

Reply:
@vhargrave - Would this code even work? I thought that `querySelectorAll` accepted a comma separated string like this:

`const pageCta = Array.from(document.querySelectorAll(
    `.section:first-of-type a.primaryCTA,
    .section:first-of-type a.cta,
    .section:first-of-type a.con-button,
    .section:first-of-type a.button`
  )).find((a) => a.href === primaryUrl.href);`

Reply:
lol, that's pretty funny